website/profile/utils.py

```diff
@@ -107,6 +111,11 @@ def serialize_contributors(contribs, node, **kwargs):
         for contrib in contribs
     ]
 
+def serialize_visible_contributors(node):
+    # This is optimized when node has .include('contributor__user__guids')
+    return [
+        serialize_user(c, node) for c in node.contributor_set.all() if c.visible
```

Comment:
Yes, but without it there's no reduction in query count.

I can't confirm this, but think that it's likely broken due to our monkey patching of QuerySets.